python/louis/__init__.py.in

```diff
@@ -44,7 +44,16 @@ import sys
 # Some general utility functions
 def _createTablesString(tablesList):
     """Creates a tables string for liblouis calls"""
-    return b",".join([x.encode("ASCII") if isinstance(x, str) else bytes(x) for x in tablesList])
+    if sys.version_info.major == 2:
+        if sys.platform == "win32":
+            return b",".join([x.decode("UTF-8") if isinstance(x, str) else bytes(x) for x in tablesList])
```

Comment:
I do not follow this code.

When there is a str (or bytes) in the tables list, it is decoded from utf-8. However, it seems utf-8 seems to be chosen arbitrarily here, how would you know that the input string is encoded as such? Furthermore, x.decode("utf-8") results in an unicode object.
Also note that something like `b"".join([u"€"])` will still output a unicode type.